tests/fmt.toml

```diff
@@ -1,9 +1,12 @@
+# This file can be used as a whole TOML file via rustfmt's --config-path option
```

Comment:
Shouldn't this file be named `rustfmt.toml`? That way it could automatically be picked up. https://github.com/rust-lang/rustfmt#configuring-rustfmt Would the nightly requirement apply as well?

Reply:
This is probably a better option. 

This file contains config that is only available in the nightly toolchain which someone using rustfmt may not want to apply or check with?

Reply:
Good point... dev laptops running stable may get constant warnings of unstable options...